### PR TITLE
[ISSUE 2830] Update junit4 to junit5 - update for common module

### PR DESCRIPTION
**Pull request by hgaol**
// Describe your PR here; eg. Fixes #issueNo

About #2830 for `shenyu-common` module.

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-common/pom.xml [hunk 1]
```diff
@@ -63,6 +63,13 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-collections4</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+<!--            <version>5.5.2</version>-->
```
Comment (haibo-duan) [inline shenyu-common/pom.xml:70]: delete annotated code.

File: shenyu-common/src/test/java/org/apache/shenyu/common/enums/WafEnumTest.java [hunk 2]
```diff
@@ -17,9 +17,9 @@
 
 package org.apache.shenyu.common.enums;
 
-import static org.junit.Assert.assertEquals;
+import org.junit.jupiter.api.Test;
 
-import org.junit.Test;
+import static org.junit.Assert.assertEquals;
```
Comment (haibo-duan) [inline shenyu-common/src/test/java/org/apache/shenyu/common/enums/WafEnumTest.java:22]: please use org.junit.jupiter.api.Assertions.assertEquals

File: shenyu-common/pom.xml (hunk 1 above)
Comment (hgaol) [reply shenyu-common/pom.xml]: resolved.

File: shenyu-common/src/test/java/org/apache/shenyu/common/enums/WafEnumTest.java (hunk 2 above)
Comment (hgaol) [reply shenyu-common/src/test/java/org/apache/shenyu/common/enums/WafEnumTest.java]: resolved.

**Comment by hgaol** [comment]
Hi @haibo-duan , resolved the comments. Please help to approve the workflow again. Thanks!

**Review by haibo-duan — APPROVED** [review]

**Review by haibo-duan — APPROVED** [review]

**Review by yu199195 — APPROVED** [review]

**Comment by yu199195** [comment]
@hgaol  thanks for contributor·， you can add my wechat  ： xixy199195

**Pull request merged**